.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["@babel/preset-flow", ["@babel/preset-env", {
+  "presets": ["@babel/preset-typescript", "@babel/preset-flow", ["@babel/preset-env", {
```

Comment:
Is there a reason that this still includes `@babel/preset-flow`?

Reply:
ah, I had it there while testing flow + ts, but it was decided against, I'll remove all the flow stuff :)